config.json

```diff
@@ -823,8 +823,8 @@
       "difficulty": 10,
       "topics": [
         "generics",
-        "higher_order_functions",
-        "nested_classes",
+        "functional_programming",
+        "classes",
```

Comment:
@eparovyshnaya what do you think about this. Should we use standard topics here?
I'm kinda ok with it.

Reply:
That's all about fitting to common metadata - it's mandatory. The other question is if this metadata rich enough to express significant pecularities. 

Both these cases are ok for me. `higher_order_functions` practically equals to `functional_programming` support, and `classes` covers `nested classes` with an appropriate context.